_posts/2024-06-06-strimzicon2024-roundup.md

```diff
@@ -0,0 +1,125 @@
+
+Speakers: Julio Turolla, Roni Silva @ Nubank
+
+In this session, Nubank engineers Julio and Roni delved into the architecture and the process of live migrating to a Strimzi setup that manages 1 trillion messages per month.
+They discussed the reasons behind choosing Strimzi, the strategies employed to minimize the impact of failures through a cell-based architecture, and the detailed mechanism of topic-by-topic live migration.
+
+#### Enhancing Kafka Topic Management in Kubernetes with the Unidirectional Topic Operator ([Recording][fede-pres]/[Slides][fede-slides])
+
+Speaker: Federico Valeri @ Red Hat
+
+This session unveils the Unidirectional Topic Operator (UTO) for Strimzi, offering Kafka administrators a seamless method to streamline topic management via Kubernetes custom resources.
+Distinguishing itself from its predecessor, the Bidirectional Topic Operator (BTO), the UTO enhances scalability and extends support to Kafka clusters operating in KRaft mode.
+
+#### Partial Multi-Tenancy on Kafka using Strimzi at LittleHorse ([Recording][littlehorse-pres]/[Slides][littlehorse-slides])
+
+Speaker: Colt McNealy @ LittleHorse
+
+The "LH Cloud Enterprise" product uses Strimzi CRD's to enable partial multi-tenancy, in which multiple applications (in their case LittleHorse clusters) share a single Kafka cluster.
+In this session Colt covered how the KafkaTopic CRD simplifies the provisioning of new LH Clusters, how they use the KafkaUser CRD to enable tenant isolation, and how they use the KafkaRebalance CRD to take advantage of Cruise Control.
+
+#### How to survive managing Strimzi in an enterprise ([Recording][axual-pres])
+
+Speaker: Richard Bosch, @ Axual BV
+
+In this session, Richard talked about experiences and insights gathered running an event streaming platform for a Tier-1 bank in The Netherlands.
+Identifying the need to answer questions like who is the data owner of our topics, which team owns these applications, and how can we contact them, led the team to develop a noteworthy enterprise topic management solution.
+
+#### Support Tiered Storage in Strimzi operated Kafka ([Recording][apple-pres]/[Slides][apple-slides])
+
+Speakers: Lixin Yao, Vrishali Bhor, Bo Gao @ Apple
+
+Lixin, Vrashali and Bo shared their experience integrating the Apache Kafka tiered storage feature with Strimzi Kafka operator and Strimzi operated Kafka clusters, and their proposal to support tiered storage feature in Strimzi natively.
+They also talked about their journey of optimizing performance for their remote storage manager implementation and the valuable insights they gained along the way.
+
+### What Next?
+
+Hopefully we'll have another StrimziCon in future, but until then we'd love to see more people using and contributing to Strimzi.
+If you're not sure where to start, here are some great ways to get involved:
+- Raising any issues you find using Strimzi
+- Fixing issues by opening Pull Requests
+- Improving documentation
+- Talking about Strimzi
```

Comment:
showing that companies are using Strimzi, maybe we could reiterate they can also have their companies to be named in the ADOPTERS and/or the website?

Reply:
Good point, added